LongWarOfTheChosen/Src/LW_FactionBalance/Classes/X2DownloadableContentInfo_LW_FactionBalance.uc

```diff
@@ -346,6 +348,33 @@ static function EditParry(X2AbilityTemplate Template)
 	PersistentEffect.SetDisplayInfo(ePerkBuff_Passive, ParryTemplate.LocFriendlyName, ParryTemplate.GetMyHelpText(), ParryTemplate.IconImage, true, , ParryTemplate.AbilitySourceName);
 	ParryTemplate.AddTargetEffect(PersistentEffect);
 }
+//Slightly modified copy pasted Robojumper's code
+static funtion UpdateCICA()
+{
+	local X2StrategyElementTemplateManager SETMgr;
+	local array<X2DataTemplate> Templates;
+	local int i;
+	SETMgr = class'X2StrategyElementTemplateManager'.static.GetStrategyElementTemplateManager();
+
+	SETMgr.FindDataTemplateAllDifficulties('CovertActionImproveComIntStaffSlot', Templates);
+	for (i = 0; i < Templates.Length; i++)

```

Comment:
Why not a `foreach`? It's a more natural construct when you don't need the index. Using an index is usually only needed if the array contains structs rather than objects.